src/main/java/ResourceDiscovery/TopicPubSubObject.java

```diff
@@ -0,0 +1,13 @@
+package ResourceDiscovery;
+
+import java.util.Map;
+
+public class TopicPubSubObject extends AssetObject {
```

Comment:
Let's schedule a f2f and discuss the function of these objects, so I can have a better understanding of their design :)